themes/default/layouts/page/get-started.html

```diff
@@ -0,0 +1,57 @@
+{{ define "hero" }}
+    {{ partial "hero.html" (dict "title" .Params.title) }}
+{{ end }}
+
+{{ define "main" }}
+    <section class="container mx-auto mt-20">
+        <div class="shape-background get-started-install-section">
+            <div class="shape-container">
+                <div class="circle-right"></div>
+            </div>
+        </div>
+        <div class="text-center mb-8 bg-violet-200 w-12 h-12 rounded-full mx-auto">
+            <p class="inline-block m-1">
```

Comment:
Suggestion from Christian

```suggestion
            <p class="flex items-center justify-center m-1">
```